proto/api/v1/tenant.proto

```diff
@@ -71,6 +72,16 @@ message PaymentDetailsUpdate {
   optional string subscription_id = 3;
 }
 
+message AcceptedTermsAndConditionsDetails {
```

Comment:
Simply TermsAndConditions ? other Message also ?

Reply:
Not sure what you mean. I try to implement this: https://github.com/metal-stack-cloud/console/issues/288  
Just save in masterdata if the user accepted the terms and when he did it.

Reply:
Shorter name of the Struct i mean `TermsAndConditions` should be enough if this has two fields `accepted` and `when`

Reply:
done